src/app/components/BalanceBox/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { useAccount } from "~/app/context/AccountContext";
+
+function BalanceBox() {
+  const { balancesDecorated } = useAccount();
+
+  const [balance, sats] = balancesDecorated.accountBalance.split(" ");
+  return (
+    <div
+      className={
+        "w-full rounded-[6px]  h-[88px] flex flex-col items-center justify-center bg-white hover:bg-gray-50 text-black dark:bg-surface-02dp dark:text-white dark:hover:bg-surface-16dp shadow transition duration-150 "
```

Comment:
- Please use named css classes (i.e. `rounded-md` instead of `rounded-[6px]`, etc)
 - Remove fixed heights (the height should be a result of individual spacing)
 - Remove hover classes
 - Remove transitions